test/services/katello/pulp/repository/docker_test.rb

```diff
@@ -42,6 +44,26 @@ def test_create
           delete_repo(@repo)
         end
 
+        def test_index_content
+          @repo.root.mirror_on_sync = true
+
+          service = Katello::Pulp::Repository::Docker.new(@repo, @master)
+          service.create
+          service2 = Katello::Pulp::Repository::Docker.new(@repo_copy, @master)
+          service2.create
+          RepositorySupport.create_and_sync_repo(@repo)
+          RepositorySupport.create_repo(@repo_copy)
+
+          @repo.index_content
+          assert_equal @repo.docker_tags.count, 248
+          service.copy_contents(@repo_copy)
+          @repo_copy.index_content(:source_repository => @repo)
+          assert_equal @repo_copy.docker_tags.count, 248
```

Comment:
we can't rely on a specific count, as the next person that runs the test in live mode may get a different result.  Something like 

assert  @repo.docker_tags.count > 0
assert_equal @repo.docker_tags.count, @repo_copy.docker_tags.count

should be sufficient.   Also  i think this test needs to be re-recorded after rebasing against master to get my vcr fixes that avoid duplicate recordings

Reply:
Rebased and test fixed